example-lazycatalog/tests/test_example.py

```
@@ -0,0 +1,57 @@
+import os
+import subprocess
+
+EXAMPLE_DIR = os.path.join(os.path.dirname(__file__), "..")
+
+
+def run_example(query: str) -> str:
+    """Run the example binary with `query` and return its stdout."""
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)